modules/gapi/test/common/gapi_video_tests_inl.hpp

```diff
@@ -131,6 +131,109 @@ TEST_P(BackgroundSubtractorTest, AccuracyTest)
     // Allowing 1% difference of all pixels between G-API and reference OpenCV results
     testBackgroundSubtractorStreaming(gapiBackSub, pOCVBackSub, 1, 1, learningRate, testNumFrames);
 }
+
+TEST_P(KalmanFilterTest, AccuracyTest)
+{
+    cv::gapi::video::KalmanParams kp(dDim, mDim, cDim, type);
+
+    //measurement vector
+    cv::Mat measure_vec(mDim, 1, type);
+    cv::randu(measure_vec, Scalar::all(-1), Scalar::all(1));
```

Comment:
I think here we should set `measure_vec` by `cv::Mat::zeros()`.

Because, see, at the first iteration, if `haveMeasure == true`, measure_vec would be filled up by `rand()` twice

Reply:
Could you tell me please what sense in asked action?

Reply:
Done.